content/en/blog/2019/alb-ingress-gateway-iks/index.md

```diff
@@ -0,0 +1,394 @@
+---
+title: Securely direct traffic from IBM Cloud Kubernetes Service Ingress to Istio Ingress Gateway
+description: Configure the IBM Cloud Kubernetes Service Application Load Balancer to direct traffic to Istio Ingress gateway with mutual TLS.
+subtitle: Configure the IBM Cloud Kubernetes Service Application Load Balancer to direct traffic to Istio Ingress gateway with mutual TLS
+publishdate: 2019-11-07
```

Comment:
does this need to be changed?

Reply:
Fixed by https://github.com/istio/istio.io/pull/5098/commits/29a21c0cef9de5b6c20d1f4eeb555c1b40be7d81.

Reply:
And also by https://github.com/istio/istio.io/pull/5098/commits/e93de4dad9242e433c149eccd1a267e8f252604d.